PINCache/PINDiskCache.h

```diff
@@ -126,6 +126,10 @@ typedef id<NSCoding> __nonnull(^PINDiskCacheDeserializerBlock)(NSData* data);
  */
 @property (assign) NSTimeInterval ageLimit;
 
+/**
+ Extension for all cache files on disk. Defaults to no extension. 
+ */
+@property (nonatomic, nullable, copy) NSString *fileExtension;
```

Comment:
Should this need to be set on init? Is there any reason you would want to change this after init?

If not, this needs a getter and setter implemented which hold onto the lock. See `willAddObjectBlock` and `setWillAddObjectBlock:`